docs/content/en/docs/clustermgmt/cluster-rebootnode.md

```diff
@@ -17,25 +17,51 @@ Rebooting a cluster node as described here is good for all nodes, but is critica
 If it does go down while running the `boots` service, the Bottlerocket node will not be able to boot again until the `boots` service is restored on another machine. This is because Bottlerocket must get its address from a DHCP service.
 {{% /alert %}}
 
-1. Cordon the node so no further workloads are scheduled to run on it:
+
+1. [Backup cluster]({{< relref "/docs/clustermgmt/cluster-backup-restore/backup-cluster" >}}) 
+
+  This ensures that there is an up-to-date cluster state available for restoration in the case that the cluster experiences issues or becomes unrecoverable during reboot.
+
+2. Verify DHCP lease time will be longer than the maintenance time, and that IPs will be the same before and after maintenance. 
+
+  This step is critical in ensuring the cluster will be viable after cluster reboot. **If this cannot be verified, do not proceed any further**
```

Comment:
Can we make this a warning for additional emphasis?
`{{% alert title="Warning" color="warning" %}}`
`{{% /alert %}}`